syntax = "proto3";

package stock_testing;

option java_package = "com.chi.ssetest.protos";
option java_outer_classname = "TestRecord";

message TestExecutionRecord {
	string jobID = 1;
	string runnerID = 2;
	string testcaseID = 3;
	string recordID = 4;
	bool isPass = 5;
	int64 startTime = 6;
	int64 endTime = 7;
	bytes paramData = 8;
	bytes resultData = 9;
	bytes exceptionData = 10;
}

message RunnerExecutionRecord {
	string jobID = 1;
	string runnerID = 2;
	int32 runCount = 3;
	int32 failureCount = 4;
	int64 runTime = 5;
}